bin/spark-evaluate.js

```diff
@@ -43,6 +48,40 @@ const createPgClient = async () => {
   return pgClient
 }
 
+const pgClient = await createPgClient()
+const cancelStuckTransactions = new CancelStuckTransactions({
+  async store ({ hash, timestamp, from, maxPriorityFeePerGas, nonce }) {
+    await pgClient.query(`
+      INSERT INTO transactions_pending (hash, timestamp, from, max_priority_fee_per_gas, nonce)
+      VALUES ($1, $2, $3, $4, $5)
+      `,
+      [hash, timestamp, from, maxPriorityFeePerGas, nonce]
+    )
```

Comment:
Is it a good idea to couple the DB schema in spark-evaluate to what `cancel-stuck-transactions` need? If the format of data stored by `cancel-stuck-transactions` changes, we will need to evolve the DB schema too.

Have you considered storing this data as JSON instead?

```js
async store ({hash, ...txData}) {
    await pgClient.query(`
      INSERT INTO transactions_pending (hash, data)
      VALUES ($1)
      `,
      [hash, txData]
    )
}

Reply:
I don't consider them coupled - `cancel-stuck-transactions` is a versioned library and if it changes the schema then we will get a dependabot PR and failing CI. I think it's great to fully SQL this, because it means we can add monitoring, plus it adds just a little bit of type safety.

If you feel strongly I can change it to JSON however, I'm not married to this method.

Reply:
> I don't consider them coupled - `cancel-stuck-transactions` is a versioned library and if it changes the schema then we will get a dependabot PR and failing CI. I think it's great to fully SQL this, because it means we can add monitoring, plus it adds just a little bit of type safety.

You convinced me 👍🏻